packages/ember-metal/lib/index.js

```diff
@@ -306,9 +306,19 @@ Ember.run = run;
 @for Ember
 @private
 */
-Ember.Backburner = Backburner;
-// this is the new go forward, once Ember Data updates to using `_Backburner` we
-// can remove the non-underscored version.
+Ember.Backburner = function() {
+  deprecate(
+    'Usage of Ember.Backburner is deprecated.',
+    false,
+    {
+      id: 'ember-metal.ember-backburner',
+      until: '2.8.0',
+      url: 'http://emberjs.com/deprecations/v2.x/#toc_ember-backburner'
+    }
+  );
+
+  return new Backburner(...arguments);
```

Comment:
I thought that was the case but I was confused by all the other uses of `...arguments` in the codebase not causing the same problem. That makes more sense now that I understand that babel will not always transpile `...arguments` to use `bind`.